src/fusor/fusion_caller_models.py

```diff
@@ -0,0 +1,239 @@
+"""Schemas for fusion callers used in translator.py"""
+
+from enum import Enum
+from typing import Literal
+
+from pydantic import BaseModel, Field
+
+
+class BaseModelForbidExtra(BaseModel, extra="forbid"):
```

Comment:
We have this class in models.py, I'd recommend only having one instance

Reply:
It also appears that you're not using this anywhere.

Reply:
You could do the following (you'd have to move it below the enum):

```python
class FusionCaller(ABC, BaseModel, extra="forbid"):
    """ABC for fusion callers"""

    type: Caller
```

and have each class below inherit this class